components/mediation-ui/org.wso2.carbon.inbound.ui/src/main/resources/web/inbound/inboundcommon.js

```diff
@@ -100,7 +100,7 @@ function inboundsave2(msg1,msg2,msg3,msg4,msg5,msg6,form){
         return false;
     }
     var subscriptionDurable = document.getElementById('transport.jms.SubscriptionDurable');
-    if (subscriptionDurable.options[subscriptionDurable.selectedIndex].value == "true" &&
+    if (subscriptionDurable != null && subscriptionDurable.options[subscriptionDurable.selectedIndex].value == "true" &&

```

Comment:
Can't we use "true".equals(subscriptionDurable.options[subscriptionDurable.selectedIndex].value) instead of == check?

Reply:
That cannot be done in javascript